spec/controllers/request_controller_spec.rb

```diff
@@ -2752,7 +2752,7 @@ def expect_redirect(status, redirect_path)
 
   it "should prevent uploads if you are not a requester" do
     @ir = info_requests(:fancy_dog_request)
-    incoming_before = @ir.incoming_messages.size
+    incoming_before = @ir.incoming_messages.count
```

Comment:
How come all these changed?

The specs still appear to pass with `#size`.

Worth explaining _why_ these changed in the commit message.

Reply:
Oh, looks like it works around isolation issues when running the full suite?

Reply:
Yea, if these were left as `size` then it would read the counter cache which might be incorrect. Will update the commit message with details.